common/src/main/java/nearsoft/skytouch/common/config/RabbitMQConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package nearsoft.skytouch.common.config;
+
+import org.springframework.beans.factory.annotation.Value;
+
+public class RabbitMQConfiguration {
+
+    @Value("${channel-config.create-products-channel-name}")
+    public static final String CREATE_PRODUCTS_CHANNEL_NAME = "createProductsChannel";
```

Comment:
This is a static final variable, and you are assigning an initialization value, and then assigning it a value from the configuration file. Are you sure it will let you do this?

Reply:
Ooo right i don't know what i was trying to do i'm going to revert this to a constant fields, is that ok?

Reply:
Ok in my new PR this fields are not present, but i'm still using constants for another fields i'm going to change that thanks